src/components/search/concept-modal/concept-modal.js

```diff
@@ -229,90 +233,50 @@ export const ConceptModalBody = ({ result }) => {
-        const loadRelatedConcepts = async (cdeId) => {
-          const formatCdeQuery = (conceptType) => {
-            return `\
-      SELECT publication-[mentions]->${conceptType}
-      FROM "/schema"
-      WHERE publication="${cdeId}"`
-          }
-          const tranqlUrl = context.tranql_url
-          const types = ['disease', 'anatomical_entity', 'phenotypic_feature', 'biological_process'] // add any others that you can think of, these are the main 4 found in heal results and supported by tranql
-          const kg = (await Promise.all(types.map(async (type) => {
-              const controller = new AbortController()
-              fetchCdesTranqlController.current.push(controller)
-              const res = await fetch(
-                  `${tranqlUrl}tranql/query`,
-                  {
-                      headers: { 'Content-Type': 'text/plain' },
-                      method: 'POST',
-                      body: formatCdeQuery(type),
-                      signal: controller.signal
-                  }
-              )
-              const message = await res.json()
-              return message.message.knowledge_graph
-          }))).reduce((acc, kg) => ({
-              nodes: {
-                  ...acc.nodes,
-                  ...kg.nodes
-              },
-              edges: {
-                  ...acc.edges,
-                  ...kg.edges
-              }
-          }), {"nodes": {}, "edges": {}})
-          const cdeOutEdges = Object.values(kg.edges).filter((edge) => edge.subject ===  cdeId)
-          return cdeOutEdges.map(
-            (outEdge) => {
-              const [nodeId, node] = Object.entries(kg.nodes).find(([nodeId, node]) => nodeId === outEdge.object)
-              return {
-                id: nodeId,
-                ...node
-              }
-            }
-          )
-        }
-
-        const loadRelatedStudies = async (cdeId) => {
-          const formatCdeQuery = () => {
-            return `\
-      SELECT publication->study
-      FROM "/schema"
-      WHERE publication="${cdeId}"`
+        const loadRelatedConceptsAndStudies = async (cdeId) => {
+          const getNodeAttribute = (node, attrName) => {
+            return node.attributes.find((attr) => attr.name === attrName)
           }
           const tranqlUrl = context.tranql_url
           const controller = new AbortController()
           fetchCdesTranqlController.current.push(controller)
           const res = await fetch(
-              `${tranqlUrl}tranql/query`,
-              {
-                  headers: { 'Content-Type': 'text/plain' },
-                  method: 'POST',
-                  body: formatCdeQuery(),
-                  signal: controller.signal
-              }
+            `${tranqlUrl}tranql/query`,
+            {
+                headers: { 'Content-Type': 'text/plain' },
+                method: 'POST',
+                body: `SELECT publication->named_thing FROM "redis:" WHERE publication="${cdeId}"`,
```

Comment:
Be careful that this could pick up (1) things that aren't concepts, or (2) concepts that we haven't loaded. For example, when I tried running this locally, I found the related concept "Rheum", which we haven't actually loaded, so trying to select it (via "Open") resulted in a `synonymousConcepts is undefined` error.

I prefer the approach we were using previously, which is to limit this query to certain types (`['disease', 'anatomical_entity', 'phenotypic_feature', 'biological_process']` in https://github.com/helxplatform/helx-ui/pull/325/files#diff-e11890aa96a2d27536af10ab83b21362d569d6467f13afd9f6792b8196bd65afL2400). I'm not sure if this will fix this problem entirely, but I think it would be helpful.

Reply:
Hmm, good point. I considered that it could possibly lead to unindexed concepts from unrestricting the types, but I didn't find in an example of such a thing happening during my testing.

Reply:
I'll note that querying against these specific types in tranql isn't going to fix the issue of getting results back that aren't indexed in dug. This can be done by checking the concept returned by tranql is linked to a study variable, but this also isn't particularly practical. There was a bug with the concept lookup mechanism (used here) which I've fixed now, so you shouldn't be seeing that error anymore.

Really, what needs to happen is the addition of a new Dug endpoint that let's you pull any documents out of the concept index by ID. Then we could do a subsequent lookup against the results returned by tranql and filter out any that aren't indexed. It's just that this feature isn't particularly prominent so the work hasn't been done.